homeassistant/components/matter/lock.py

```diff
@@ -120,9 +120,6 @@ async def async_unlock(self, **kwargs: Any) -> None:
 
     async def async_open(self, **kwargs: Any) -> None:
         """Open the door latch."""
-        # optimistically signal unlocking to state machine
```

Comment:
I think we should not remove it completely but change it to 

self._attr_is_opening = True

and then add a timer to reset the state if no update comes in.
I need to do another PR for lock today so I can adjust this as well

Reply:
But it will be stuck in opening state, right?
Should we remove this transitioning state until Nuki updates the state to avoid issue for now?